py/torch_tensorrt/fx/utils.py

```diff
@@ -37,6 +41,11 @@ def torch_dtype_to_trt(dtype: torch.dtype) -> TRTDataType:
         return trt.int8
     elif dtype == torch.int32:
         return trt.int32
+    elif dtype == torch.int64:
+        _LOGGER.warn(
+            "Detected Int64 Input, Casting to Int32 for TRT Engine Compatibility"
+        )
+        return trt.int32
```

Comment:
Could potentially be enabled by a flag similar to the TorchScript path (like `truncate_long_and_double`), instead of a warning, if this is not the desired default behavior